scss/civicrm/contact/pages/_task-label.scss

```diff
@@ -0,0 +1,11 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+.CRM_Contact_Form_Task_Label {
+  .crm-contact-task-mailing-label-form-block {
+    padding-top: 20px;
```

Comment:
you have the same thing going on in
```scss
.crm-contact-task-delete-form-block {
  padding-top: 20px;
}
```
what is this for? why is it duplicated?

Reply:
This is a different page containing different form blocks. see below,
**Delete Task page**
![image](https://user-images.githubusercontent.com/26058635/37083557-df820ffe-2215-11e8-8a0a-6b182a7c2916.png)
**Print Label task page**
![image](https://user-images.githubusercontent.com/26058635/37083514-bf65774c-2215-11e8-9ec6-85cac2d471fe.png)

Reply:
and there is no generic rule to be found?

Reply:
Nope, there are different tasks for forms and we just need to target this specific ones. not to touch others.
There is `.crm-form-block` but, it's too generic and can break other forms.